pkg/kubefed2/federate/enable.go

```diff
@@ -247,6 +255,7 @@ func CreateResources(config *rest.Config, resources *typeResources) error {
 	if err != nil {
 		return fmt.Errorf("Error creating FederatedTypeConfig %q: %v", concreteTypeConfig.Name, err)
 	}
+	write(fmt.Sprintf("federatedtypeconfig.core.federation.k8s.io/%s created\n", concreteTypeConfig.Name))
```

Comment:
It might be helpful to have the namespace in this message, since you may not have supplied it as an arg.